tools/pyclient/dev/dev.py

```diff
@@ -0,0 +1,85 @@
+#///////////////////////////////////////////////////////////////////////////////
+# FILE: dev.py
+# AUTHOR: David Ruvolo
+# CREATED: 2023-05-22
+# MODIFIED: 2023-07-21
+# PURPOSE: development script for initial testing of the py-client
+# STATUS: ongoing
+# PACKAGES: pandas
+# COMMENTS: Designed to interact with the schema "pet store".
+#///////////////////////////////////////////////////////////////////////////////
+
+import pandas as pd
+
+from tools.pyclient.src.molgenis_emx2_pyclient import Client
+
+# connect and sign in
+db = Client('https://emx2.dev.molgenis.org/')
+db.sign_in('', '')
+
+# check status
+db.status()
+
+# get data
+data = db.get(schema='', table='') # run without specifying target

```

Comment:
what do I get in this case? I propose you also start adding a section in the /docs ?

If I don't think to hard I would expect a huge age result with all tables contents of all schemas?

Reply:
This will throw an error informing the user to specify a schema and table.